homeassistant/components/homematicip_cloud/climate.py

```diff
@@ -48,6 +50,8 @@ class HomematicipHeatingGroup(HomematicipGenericDevice, ClimateDevice):
     def __init__(self, home: AsyncHome, device) -> None:
         """Initialize heating group."""
         device.modelType = 'Group-Heating'
+        if device.actualTemperature is None:
+            self._simple_heating = _get_first_heating_thermostat(device)
```

Comment:
We should never make an attribute's existence optional. What we can do is set an attribute to `None` to turn off that feature, permanently or temporary. All attributes of the instance need to be set to something in the init method.